src/scss/_general.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: linear-gradient($primary, $secondary, $primary);
+  position: relative;
+}
+
+.img-fluid {
+  width: 100%;
+}
+
+.feed_page {
+  background-color: $profile;
+  padding-bottom: 30px;
+  justify-content: center;
+  justify-items: center;
+}
+
+.feed_posts {
+  width: 70%;
+  margin: 20px auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.thumbnail {
```

Comment:
dette er alt for mye css igjen. Jeg tror det kunne være lurt for deg å lage en enkel side med kun bootstrap cdn og absolutt ingen css eller scss